keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -623,6 +630,8 @@ and add `MatchLabelSelectors` on their PodAffinity/PodAntiAffinity.
 kube-apiserver will ignore `MatchLabelSelectors` in PodAffinity/PodAntiAffinity,
 and thus, kube-scheduler will also do nothing with it.
 
+But, we leave the existing `MatchLabelSelectors` and `LabelSelector` created from `MatchLabelSelectors` as it is.
```

Comment:
Actually, I have an additional question: IIUC, PodAffinity/PodAntiAffinity is immutable, right? So once I create pod, I can no longer change that, right?

If the above is true, we should clarify that:
- if FG is disabled, MatchLabelSelectors will be dropped for newly created pods and for existing pods
- if FG is disabled, MatchLabelSelectors and changes to LabelSelector  propagated from it will remain as they are for already existing pods

Reply:
Yes, right. Sure, let me clarify more on the KEP.

Reply:
https://github.com/kubernetes/enhancements/pull/4069/commits/c6413055b5da47753b7d50215e6cec124788fffe

If FG is disabled, 
- for Pod creation, pods with MatchLabelSelectors will be rejected in validation.
- for Pod update (= for existing Pods), pods with MatchLabelSelectors will not be rejected, it'll remain as it is. 

Does it sound good?